service/controller/chartconfig/v7/key/key.go

```diff
@@ -23,6 +25,22 @@ func ConfigMapNamespace(customObject v1alpha1.ChartConfig) string {
 	return customObject.Spec.Chart.ConfigMap.Namespace
 }
 
+func CordonReason(customObject v1alpha1.ChartConfig) string {
+	val, ok := customObject.Annotations[annotation.CordonReasonAnnotationName]
```

Comment:
When you do it like that you can simply return the value and drop the rest. We do that in different places already. See https://github.com/giantswarm/cluster-operator/blob/master/service/controller/clusterapi/v17/key/common.go. 

```suggestion
	return customObject.Annotations[annotation.CordonReasonAnnotationName]
```

Reply:
I also change the code as `customObject.GetAnnotation()` to following the other codes.